modules/programs/zsh.nix

```
@@ -622,6 +632,7 @@ in
           # https://github.com/zsh-users/zsh-syntax-highlighting#faq
         ''
           source ${cfg.syntaxHighlighting.package}/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh
+          ZSH_HIGHLIGHT_HIGHLIGHTERS+=(${lib.concatStringsSep " " cfg.syntaxHighlighting.highlighters})
```

The values should be escaped:
```suggestion
          ZSH_HIGHLIGHT_HIGHLIGHTERS+=(${lib.concatStringsSep " " (map lib.escapeShellArg cfg.syntaxHighlighting.highlighters)})
```

Tests will probably need to be adjusted, and obviously make sure this still works!

---

Wow, IIRC I escaped it for the first try and it didn't work. But now I tried it again and no issues, thanks!